pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -426,7 +426,7 @@ type StatefulSetSettings struct {
 }
 
 type EtcdStatefulSetSettings struct {
-	ClusterSize  int                          `json:"clusterSize,omitempty"`
+	ClusterSize  *int32                       `json:"clusterSize,omitempty"`
```

Comment:
What's the point of this change?

Reply:
To allow distinguishing size unset and size set to zero.

Reply:
While yes, technically this is correct, I meant: is this what we need, what's the point of having a Cluster of 0 etcds?

Reply:
I can't tell exact scenario. But sometimes users want to scale things down to zero. As an analogy - replicas count in deployments are also pointers and sometimes users scale them down to zero.

Reply:
Why `int32` and not `int`?